packages/perseus/src/widgets/interactive-graphs/utils.ts

```diff
@@ -81,7 +82,7 @@ export function isUnlimitedGraphState(
 export function replaceOutsideTeX(mathString: string) {
     // All the information we need is in the first section,
     // whether it's typed as "blockmath" or "paragraph"
-    const firstSection = SimpleMarkdown.parse(mathString)[0];
+    const firstSection = parse(mathString)[0];
```

Comment:
Not making any changes here, just updating the export. This whole function will be updated in a different PR.